### PR TITLE
Added to README, refactor, more flexibility

### PR DESCRIPTION
**Pull request by swiftlee**
Changes:

**README.md**
Minor changes to the README to give better instruction on how to run the program.

**Program.cs**

- Refactored usages of "var" and boolean expressions to be more consistent and readable.
- Allow base path to include backslashes.
- Allow other URLs besides the /Books endpoint.
- Added console prompt for entering your own URLs.

**.gitignore**
Added support for JetBrains Rider so that .idea/ directory is ignored.

### CODE REVIEW

File: Program.cs [hunk 1]
```diff
@@ -8,86 +8,79 @@
...
+            _basePath = Console.ReadLine()?.Replace('\\', '/').Replace(":\\", "://");
 
-            if(string.IsNullOrWhiteSpace(BasePath))
-            {
-                throw new ArgumentNullException(BasePath, "Invalid path");
+            if (IsNullOrWhiteSpace(_basePath)) {
+                throw new ArgumentNullException(_basePath, "Invalid path");
             }
+            
+            Console.Write("Enter the URL to download from:");
```
Comment (felipegiacomozzi) [inline Program.cs]: Wouldn't be best if the asked the url as a optional, leaving books as default?
Or at least inform the possible options to the user.

What do you think?

**Review by felipegiacomozzi — CHANGES_REQUESTED** [review]
Set the Books URL as default and/or list other options for URL

**Comment by swiftlee** [comment]
Agreed! I think I intended for that actually but somehow it didn't make it in there... good point!

File: Program.cs (hunk 1 above)
Comment (swiftlee) [reply Program.cs]: Agreed! I think I intended for that actually but somehow it didn't make it in there... good point!

**Review by felipegiacomozzi — COMMENTED** [review]
Need to set the default value as Books

**Comment by swiftlee** [comment]
> Need to set the default value as Books

Yes! This should already be the case.

**Pull request merged**